src/base_alloc/base_alloc.h

```diff
@@ -21,7 +21,9 @@ typedef struct umf_ba_pool_t umf_ba_pool_t;
 umf_ba_pool_t *umf_ba_create(size_t size);
 void *umf_ba_alloc(umf_ba_pool_t *pool);
 void umf_ba_free(umf_ba_pool_t *pool, void *ptr);
+void umf_ba_init(void);
 void umf_ba_destroy(umf_ba_pool_t *pool);
+void umf_ba_destroy_global(void);
```

Comment:
nit: should;t this be in base_alloc_global.h? (umf_ba_init should also be there and be named umf_ba_global_init probably)

Reply:
Done. `umf_ba_init()` is OK.

Reply:
`umf_ba_init()` is removed.
Done.